packages/twenty-front/src/modules/favorites/favorite-folder-picker/components/FavoriteFolderMenuItemMultiSelect.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled';
+import { MenuItemMultiSelect } from '@ui/navigation/menu-item/components/MenuItemMultiSelect';
+
+const StyledNoGapMenuItem = styled(MenuItemMultiSelect)`
+  & > div {
```

Comment:
we should avoid styling child component from the outside. This is likely a standard behavior of the MenuItemMultiSelect

Reply:
Should I introduce a prop on MenuItemMultiSelect to address the gap? I don't think creating a separate component in this case is a good idea